### ISSUE TITLE
compiler-version should reject files without versions

### ISSUE THREAD

**Issue by fvictorio** [protofire/solhint#130]
If `compiler-version` is enabled, then a file like

```solidity
contract Foo {}
```

shouldn't pass that rule because there's no pragma with the solidity version.

**Comment by pablofullana** [comment]
@fvictorio shall we close this one then?

**Comment by fvictorio** [comment]
Closed by #172.
